### DISCUSSION TITLE
Google Product Taxonomy

### DISCUSSION THREAD

**Discussion by FJPlusOneX** [saleor/saleor#12870] [General] [answered]
Hi there,

I am looking for a method to assign a Google Product Category to each product in Saleor. Here is a link to understand what the Google Product Category is https://support.google.com/merchants/answer/6324436?hl=en

Essentially this allows services like Google Merchant Center and Meta Catalog to categorise and serve products to users who are interested in those product categories.

Is there a solution in Saleor where Google's product taxonomy can be applied to each product? I've attached the full product taxonomy here:

[taxonomy-with-ids.en-US.xls](https://github.com/saleor/saleor/files/11518207/taxonomy-with-ids.en-US.xls)

Any knowledge on this would be much appreciated.

With thanks,
F

**Comment by mociepka** [comment, marked as answer]
Hi,

You can recreate a Google product category that fits your business case as Saleor Categories. Saleor has a tree structure category system, so you are not restricted.

When you have your category structure in place, you can add `google_product_category` as a [metadata](https://docs.saleor.io/docs/3.x/developer/metadata):


https://github.com/saleor/saleor/assets/1870738/94aa3c2e-7a42-4b7b-9a96-83abc90b6ef8



and get it by the:
```graphql
{
  product(id:"UHJvZHVjdDo4OQ==", channel: "default-channel") {
    id,
    name,
    category {
      name
      google_category_product: metafield(key: "google_category_product")
    }
  }
}
```

**Comment by FJPlusOneX** [comment]
Incredible, thank you very much.
